enterprise/server/remote_execution/execution_server/execution_server.go

```diff
@@ -66,6 +67,12 @@ var (
 	enableActionMerging               = flag.Bool("remote_execution.enable_action_merging", true, "If enabled, identical actions being executed concurrently are merged into a single execution.")
 )
 
+const (
+	// gRPC header containing the serialized RemoteSnapshotOptions.
+	// Keep in sync with enterprise/server/workflow/service.go
+	snapshotOptionsHeader = "x-buildbuddy-remotesnaphotopts-bin"
```

Comment:
can this go in platform / be an x-buildbuddy-platform.* header?

Reply:
That feels a bit brittle, because it could potentially affect the routing key. Platform overrides don't technically affect the routing key today (and don't get included in various platform hashes), but I feel that's a bug that we should eventually fix.